docs/administration/configuration/configuring-external-access/index.md

```diff
@@ -0,0 +1,136 @@
+# Configuring external access
+
+REANA manages external access with the use of Kubernetes [Ingresses](https://kubernetes.io/docs/concepts/services-networking/ingress/).
+The Helm chart automatically deploys Traefik as an [Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/) and creates the necessary Ingress resources, but this can be changed if you wish to deploy your own Ingress Controller or your own Ingresses.
+
+## Configuring generated Ingresses
+
+REANA creates a single Ingress object by default, which can be customized with the following [Helm values](https://github.com/reanahub/reana/tree/master/helm/reana):
+
+- `ingress.enabled` enables or disables the creation of Ingresses
+- `ingress.annotations` is used to define which custom annotations should be applied to the Ingress resource
+- `ingress.tls` can be used to configure a TLS certificate (see [Configuring TLS certificates](../configuring-tls-certificates/))
+
+You can also specify additional Ingresses with the `ingress.extra` Helm value.
+Each additional Ingress can be configured in the same way as the default one, with the addition of `ingress.extra[].name`.
+
+As an example, this is how you would configure Traefik by creating two Ingresses, one to handle HTTPS traffic and the other to redirect HTTP requests to HTTPS.
```

Comment:
... to HTTPS:

(indicating that the topic is not finished and that the code snippet below will exemplify it)